lib/tasks/mfa_policy.rake

```diff
@@ -32,4 +32,23 @@ namespace :mfa_policy do
       print format("\r%.2f%% (%d/%d) complete", i.to_f / total_users * 100.0, i, total_users)
     end
   end
+
+  # This task is meant to be run one week prior to MFA Phase 3 launch day - send out on Aug 8, 2022
+  # For more information on the MFA Phase 3 rollout, refer to this RFC:
+  # https://github.com/rubygems/rfcs/pull/36/files#diff-3d5cc3acc06fe7e9150fdbfc43399c5ad42572c122187774bfc3a4857df524f1R69-R85
+  # rake mfa_policy:reminder_enable_mfa
+  desc "Send email reminder to users who will have MFA enforced about impending MFA Phase 3 rollout"
+  task reminder_enable_mfa: :environment do
+    # users who own at least one gem with 180,000,000 downloads or more with weak or no MFA
+    users = User.joins(rubygems: :gem_download).where("gem_downloads.count > 180000000").where(mfa_level: %w[disabled ui_only])
+    total_users = users.count
+    puts "Sending #{total_users} MFA announcement email"
```

Comment:
The puts/print is creating a bit of noise in CI, is there a way to silence these logs?

<img width="877" alt="Screen Shot 2022-08-04 at 10 33 42 AM" src="https://user-images.githubusercontent.com/42748004/182873658-777e872e-0919-4bec-8f1a-45e0c0ce4f83.png">

Reply:
Hm, I'm not sure 🤔. The puts/print is happening because we have an integration test that triggers the rake task which contains the puts/print in its codepath. 

Alternatively, I can refactor the mailer tests to be unit tests. They're currently set up as integration tests, testing both the rake task and the mailer itself. If I change it to be unit tests, it will bypasses the rake puts/print logging from the rake tasks.

Reply:
Mhm, and if you change them to unit tests, it will bypass the rake puts/print logging from the rake tasks because the task wouldn't be run at all? It'll be nice to still test the rake task imo. 

The only solution I can think of that would work atm is only puts/print if the env isn't `test`. Alternatively, an issue can be opened so we can address this separately.

Reply:
Yea the glaring tradeoff of integration vs. unit is we will lose test coverage on the rake tasks. Good point re: checking for env!

Reply:
Alternatively you may be able to use minitest's `capture_io`: https://www.rubydoc.info/gems/minitest/5.9.0/Minitest/Assertions#capture_io-instance_method.

Reply:
Oh neat! Thanks for the nudge in the right direction, David 🙇‍♀️. I'll dig into that.

Reply:
That did the trick! Thanks @deivid-rodriguez 🙌